packages/react-textarea/src/stories/TextAreaSize.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { makeStyles, shorthands } from '@griffel/react';
+import { useId } from '@fluentui/react-utilities';
+import { Label } from '@fluentui/react-label';
+import { TextArea } from '../TextArea';
+
+const useStyles = makeStyles({
+  container: {
+    '> div': { ...shorthands.padding('20px') },
+    '& label': { display: 'block', marginBottom: '10px' },
+  },
+});
+
+export const Size = () => {
+  const smallId = useId('textarea-small');
+  const mediumId = useId('textarea-medium');
+  const largeId = useId('textarea-large');
+  const styles = useStyles();
+
+  return (
+    <div className={styles.container}>
+      <div>
+        <Label htmlFor={smallId}>Small TextArea.</Label>
+        <TextArea id={smallId} size="small" placeholder="Small TextArea" />
```

Comment:
A small thing, but we shouldn't have placeholder text duplicate the label. I'd suggest either removing it entirely, or sticking with the "placeholder text" convention from other examples

Reply:
For Input I got rid of it entirely.